packages/neon/neon/lib/neon.dart

```diff
@@ -17,6 +17,8 @@ import 'package:package_info_plus/package_info_plus.dart';
 import 'package:provider/provider.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
+late final String neonUserAgent;
```

Comment:
can we make this private/hidden?

I don't it needs to be part of our api.

Reply:
You mean annotating with `@internal`? Because making it private wouldn't work as we still need to import it throughout the app

Reply:
Ah yeah I didn't think about this.

Yes in this case `@internal` might make the most sense